docs/development/release_management.rst

```diff
@@ -166,8 +171,8 @@ Release Process
 11. Step through the signing ceremony for the ``Release``
     file(s) (there may be multiple if Tor is also updated along
     with the SecureDrop release).
-12. Coordinate with the Infrastructure team to put signed Debian packages on
-    ``apt-qa.freedom.press``.
+12. Coordinate with the Infrastructure team to put signed Debian packages, including
+    Tor and kernel packages if applicable,  on ``apt-qa.freedom.press``.
```

Comment:
we might want to be more specific here, we did add the metapackage

Reply:
i.e. "if you add a new securedrop-grsec-* metapackage, ensure that you add the corresponding linux-image-* file"

Reply:
:heavy_check_mark: